docs/reference.rst

```diff
@@ -165,6 +165,13 @@ A valid manifest file must have the file extension ``.raucm``.
   List of hooks enabled for this bundle.
 
 
+**[handler] section**
+
+``filename``
+  Handler script path name, relative to the bundle content. Used to fully
+  replace default update process.
```

Comment:
Seems like I missed args options when going through the source code. Documentation about it has been added in the next commit.